distribution/tools/plugin-cli/src/test/java/org/elasticsearch/plugins/cli/InstallPluginActionTests.java

```diff
@@ -124,13 +129,17 @@ public class InstallPluginActionTests extends ESTestCase {
     private MockTerminal terminal;
     private Tuple<Path, Environment> env;
     private Path pluginDir;
+    private ClassReadersProvider classReadersProvider;
+    private NamedComponentScanner namedComponentScanner;
 
     private final boolean isPosix;
     private final boolean isReal;
     private final String javaIoTmpdir;
 
     @SuppressForbidden(reason = "sets java.io.tmpdir")
     public InstallPluginActionTests(FileSystem fs, Function<String, Path> temp) {
+        assert "false".equals(System.getProperty("tests.security.manager")) : "-Dtests.security.manager=false has to be set";
```

Comment:
I think using just assert looks better. this is not a real assertion but a hint when running from IDE
with assumeTrue it would look confusing
assumeTrue("-Dtests.security.manager=false has to be set", "false".equals(System.getProperty("tests.security.manager"));